users/relaying.rst

```diff
@@ -27,7 +27,30 @@ devices.
 Running Your Own Relay
 ----------------------
 
-To run a relay of your own, download the latest release of the `relay server <https://github.com/syncthing/relaysrv/releases>`__
+Syncthing relies on a network of community-contributed relay servers. Anyone can
+run a relay server, and it will automatically join the relay pool and be
+available to Syncthing users. The current list of relays can be found at
+https://relays.syncthing.net.
+
+To run a relay of your own, you will first need a server. Running a relay from a
+home computer is not recommended: the relay ideally needs to be running 24/7,
+and home internet connections often have poor upload bandwidth.
+
+The recommended specifications for a relay are:
+
+========= ==============
+CPU       At least 1GHz
+RAM       At least 256MB
+Bandwidth At least 1MBit/s in and out, the higher the better
+Traffic   At least 1TiB/month, the more the better
+========= ==============
```

Comment:
CPU and RAM were from looking at relaysrv's usage on my relays. Bandwidth was from the minimum rate limits used by current relays, and traffic is bandwidth + maths.